metagpt/provider/base_llm.py

```diff
@@ -22,6 +22,7 @@
     wait_random_exponential,
 )
 
+from metagpt.config2 import config
```

Comment:
Thank you very much for your contribution.

The current design does not introduce global config in base_llm, but passes it in from the outside.

Reply:
thank you for pointing that out.
I have changed the code by using BaseLLM.config.stream other than from global.